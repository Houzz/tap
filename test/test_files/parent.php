<?php

class A {
    public function foo() {
        echo "hello\n";
    }
}

class B extends A {
    public function foo() {
        parent::foo();
        parent::bar();  // error
    }
}

