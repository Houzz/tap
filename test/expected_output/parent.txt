12 E_FUNCTION_NOT_FOUND
